### PR TITLE
Added minimal split bug test

### PR DESCRIPTION
**Pull request by shreyas-kowshik**

### CODE REVIEW

File: test/Logical/TransformationsTest.jl [hunk 1]
```diff
@@ -148,6 +148,22 @@ end
     @test n1.children[1] == n2.children[1]
 end
 
+"""
+This reproduces the bug that for multiple depths, the result is the same
+"""
```
Comment (khosravipasha) [inline test/Logical/TransformationsTest.jl:153]: I think we cannot add tripple quote comments before "testtset" here. Error from the travis log:

```
Error During Test at /home/travis/build/Juice-jl/LogicCircuits.jl/test/Logical/TransformationsTest.jl:151
  Got exception outside of a @test
  LoadError: cannot document the following expression:
.
.
.
.

 '@testset' not documentable. See 'Base.@__doc__' docs for details.
```

Comment (shreyas-kowshik) [reply test/Logical/TransformationsTest.jl]: Updated in latest commit.

**Comment by guyvdbroeck** [comment]
Is this bug still relevant? did we add a test for it in some other way?

**Comment by shreyas-kowshik** [comment]
No it is not relevant now. The implementation was changed for split and has been added to the repository.

**Pull request closed without merging**